pkgs/servers/keycloak/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchzip, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  name    = "keycloak-v${version}";
```

Comment:
Versions cannot begin with a letter. You need `name = keycloak-${version}`, or better `pname = "keycloak"`

Reply:
Fixed, thank you.